department-of-veterans-affairs/abd-vro
# Add new classification mappings for Redesigned List

## What was the problem?
This PR includes the classification mappings for dropdown conditions in the new list of conditions. 

Associated tickets or Slack threads:
- [CC #327](https://app.zenhub.com/workspaces/contention-classification-640a24db5dcd08457cebbc79/issues/gh/department-of-veterans-affairs/vagov-claim-classification/327)

## How does this fix it?[^1]


## How to test this PR
- From domain-cc directory run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
